### DISCUSSION TITLE
Parametrierter Aufruf der Attribute in der Editiermaske

### DISCUSSION THREAD

**Discussion by thomaskofler** [e-netze/webgis-community#129] [General]
Es sollte beim parametrierten Aufruf ermöglicht werden, ein Editierthema inkl. Abfrage z.B. Objekt-ID aufzurufen, dabei soll sich die Editiermaske (Attribute) automatisch öffnen.

**Comment by gis-eni** [comment]
Sollte bis auf das Öffnen der Editiermaske gehen. Bisher muss der Anwender dann noch auf "neues Objekt erstellen" klicken.
Das die Maske gleich aufgeht kommt erst mit den neuen Editierfunktionen mit WebGIS 6

**Reply by thomaskofler** [reply]
Ok, bis zu "neues Objekt erstellen" habe ich es parametriert.
